File: URI/C _ C++/Beginner/1043.c
#include <stdio.h>

int main(void)
{
    float a,b,c,area;

    scanf("%f %f %f", &a,&b,&c);

    if(a<b+c && b<c+a && c<a+b) {
        area=a+b+c;
        printf("Perimetro = %.1f\n", area);
    }

    else {
        area = .5*c*(a+b);
        printf("Area = %.1f\n", area);
    }

    return 0;
}

File: URI/Py/1043.py
a,b,c = map(float, input().split())

d = [a,b,c]
d.sort()

if d[0]+d[1] > d[2]:
    print("Perimetro = %.1f" %sum(d))
else:
    d = (a + b) * c / 2
    print("Area = %.1f" %d)